playground/terraform/infrastructure/cloudbuild-manual-setup/02.builders/triggers.tf

```diff
@@ -77,5 +77,29 @@ resource "google_cloudbuild_trigger" "playground_to_gke" {
 
   filename = "playground/infrastructure/cloudbuild/cloudbuild_pg_to_gke.yaml"
 
+  service_account = data.google_service_account.cloudbuild_sa.id
+}
+
+resource "google_cloudbuild_trigger" "playground_examples_cd" {
+  name     = var.examples_cd_trigger_name
+  project  = var.project_id
+
+  description = "Runs CD pipeline steps for Playground Examples"
+
+  source_to_build {
```

Comment:
Can we use github here instead of source_to_build?

Reply:
Since we are doing CD as manual, it should be the way it is, open for discussion though